src/esphomelib/log_component.h

```diff
@@ -17,6 +17,22 @@
 
 ESPHOMELIB_NAMESPACE_BEGIN
 
+/** Enum for logging UART selection
+ *
+ * Advanced configuration (pin selection, etc) is not supported.
+ */
+enum UARTSelection {
+  ESPHOMELIB_UART0 = 0,
```

Comment:
For enums, I prefer to prefix them with the enum name (I think this is google style too, but not sure).

So `UART_SELECTION_UART0 = 0,`

(Or in this case mostly just so that I don't have to change this to `ESPHOME_UART0` in https://github.com/esphome/esphome-core/pull/494 again 😜 )

Reply:
Can do! I think I copied it from the log stuff - ESPHOMELIB_LOG_X etc.